events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+    //     }
+    // }
+
+    function getQuestion() {
+        Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['NEXT_QUESTION_SFX']);
+        try {
+            var triviaURL = "https://opentdb.com/api.php?amount=1";
+            if (type) {
+                triviaURL = triviaURL + "&type=" + type;
+            }
+            if (difficulty) {
+                triviaURL = triviaURL + "&difficulty=" + difficulty;
+            }
+            if (category) {
+                triviaURL = triviaURL + "&category=" + category;
+            }
+            request(triviaURL, function (error, data) {
+                if (!error) {
+                    tablet.emitScriptEvent(JSON.stringify(data.results[0]));
+                    triviaData = data.results;
+                }
+            });
+        } catch (err) {
+            print("Could not get domain data using userData domainAPIURL");
+        }
+    }
+
+    function shuffle(array) {
+        var currentIndex = array.length, temporaryValue, randomIndex;
+      
+        while (currentIndex !== 0) {
+            randomIndex = Math.floor(Math.random() * currentIndex);
+            currentIndex -= 1;
+            temporaryValue = array[currentIndex];
+            array[currentIndex] = array[randomIndex];
+            array[randomIndex] = temporaryValue;
+        }
+        return array;
+    }
+
+    function showQuestion() {
+        clearBoard();
+        Script.setTimeout(function() {
+            choiceTexts.forEach(function(choice) {
+                Entities.callEntityServerMethod(choice, "textUpdate", ["", true]);
+            });
+            var formattedQuestion = htmlEnDeCode.htmlDecode(triviaData[0].question);
+            // var formattedQuestion = triviaData.question;
+            Entities.callEntityServerMethod(questionText, "textUpdate", [formattedQuestion, true]);
+            Entities.callEntityServerMethod(answerText, "textUpdate", ["", false]);
+        }, WAIT_TO_SHOW_QUESTION);
+    }
+
+    function showAnswers() {
+        if (triviaData[0].type === "boolean") {
+            Entities.callEntityServerMethod(choiceTexts[0], "textUpdate", ["",false]);
+            Entities.callEntityServerMethod(choiceTexts[1], "textUpdate", ["True", true]);
+            Entities.callEntityServerMethod(choiceTexts[2], "textUpdate", ["False", true]);
+            Entities.callEntityServerMethod(choiceTexts[3], "textUpdate", ["", false]);
+
+            currentChoices = [];
+            currentChoices.push("True");
+            currentChoices.push("False");
+            lights.forEach(function(light) {
+                var lightName = Entities.getEntityProperties(light, 'name').name;
+                if (lightName.indexOf("Green") !== -1) {
+                    Entities.callEntityServerMethod(light, "lightsOff");
+                } else if (lightName.indexOf("Blue") !== -1) {
+                    Entities.callEntityServerMethod(light, "lightsOff");
```

Comment:
DRY up this code by using a logical OR in one conditional rather than a branching conditional

Reply:
fixed